src/Profile.php

```diff
@@ -66,6 +66,8 @@ public static function fromUuid($uuid, ApiClient $apiClient = null)
             }
         }
 
+        $profile->history = $apiClient->uuidHistory($uuid);
```

Comment:
How about adding an exception to the __get() method; if the $key variable equals 'history' and no $history property has been set yet, hit the api. If it has been set just return it.

This way you get the same functionality, but only hit the api if the user requests it. And no need for extra functions. 

What do you think? 😄